creator/cxi_converter.py

```diff
@@ -0,0 +1,211 @@
+            entry = creator.create_entry_group(definition='NXptycho',
+                                               entry_index=n,
+                                               experiment_description="basic",
+                                               title='test_experiment')
+            instrument = creator.create_instrument_group(h5parent=entry,
+                                                         name=f"{data_dict(n)['source_name']} {data_dict(n)['instrument_name']}")
+            creator.create_beam_group(h5parent=instrument,
+                                      incident_beam_energy=data_dict(n)["energy"],
+                                      energy_units='eV')
+            detector = creator.create_detector_group(h5parent=instrument,
+                                                     data=data_dict(n)["data"],
+                                                     data_units='counts',
+                                                     distance=data_dict(n)["distance"],
+                                                     distance_units='m',
+                                                     x_pixel_size=data_dict(n)["x_pixel_size"],
+                                                     y_pixel_size=data_dict(n)["y_pixel_size"],
+                                                     pixel_size_units='um')
+            creator.create_data_group(h5parent=entry, signal_data='data')
+            transformation = creator.create_transformation_group(h5parent=detector)
+            # create transformation axes
+            creator.create_axis(transformation=transformation,
+                                axis_name='x_translation',
+                                value=34,
+                                transformation_type='translation',
+                                vector=np.array([1, 0, 0], dtype=float),
+                                offset=np.zeros(3, dtype=float),
+                                units="m",
+                                depends_on=".")
+            creator.create_axis(transformation=transformation,
+                                axis_name='y_translation',
+                                value=34,
+                                transformation_type='translation',
+                                vector=np.array([0, 1, 0], dtype=float),
+                                offset=np.zeros(3, dtype=float),
+                                units="m",
+                                depends_on="x_translation")
+            creator.create_axis(transformation=transformation,
+                                axis_name='z_translation',
+                                value=34,
+                                transformation_type='translation',
+                                vector=np.array([0, 0, 1], dtype=float),
+                                offset=np.zeros(3, dtype=float),
+                                units="m",
+                                depends_on="y_translation")
```

Comment:
I'm assuming these transformations are placeholders because none of them make sense.

Reply:
That is correct, I do not have actual values for these axes. So I removed the value unless they actually contain any real data such as for the fine sample translations, where I link to the positioner group.